src/Marr.Data/Marr.Data.csproj

```diff
@@ -144,7 +144,7 @@
     <PreBuildEvent>
     </PreBuildEvent>
   </PropertyGroup>
-  <Import Project="$(SolutionDir)\.nuget\nuget.targets" />
+  <Import Project="$(SolutionDir)\.nuget\NuGet.targets" />
```

Comment:
This stage is case sensitive, so the case of the variable and the file must match.